x/subscription/spec/03_messages.md

```diff
@@ -0,0 +1,94 @@
+<!--
+order: 3
+-->
+
+# Messages
+
+## MsgCreatePlan
+
+Merchant or payment provider create plans.
+
+```protobuf
+message MsgCreatePlan {
+  string title;
+  string description;
+  Decimal price,  // price in fiat currency show to user
+  string fiat_currency,  // fiat currency name
+  string token_denom,  // denomination of token
+  Decimal conversion_rate,  // conversion rate from fiat currency to token, maintained by owner
```

Comment:
@christopher-crypto maybe good to have a possibility to specify 1-to-1 plans which will be automatically deducted at begin block (without the plan owner submitting collect payment msg)

Reply:
or simplify to only 1-to-1 rates (only specify denom, but no conversions)

Reply:
for "begin block" collection, advantage is no extra fees

Reply:
now it supports:
- automatically collect in begin block once for each period
- merchant retry collection manually at any time